src/Writer/ArrayWriter.php

```diff
@@ -9,6 +9,8 @@
  */
 class ArrayWriter implements WriterInterface
 {
+    use WriterTemplate;
```

Comment:
I can imagine a `NullWriter` as a real writer for test purposes. I think template is a good name, because it is actually a template. Some alternatives I can think of:

- NoopWriter
- BaseWriter

However I think Template says the most about the purpose of that trait.